serve/mlc_serve/engine/staging_engine.py

```diff
@@ -209,6 +211,7 @@ def step(self) -> InferenceStepResult:
 
                 with structlog.contextvars.bound_contextvars(**state.contextvars):
                     if seq_output.error is not None:
+                        LOG.info("Error")
```

Comment:
Remove it or add more descriptive message.

Reply:
Removed.